docs/reference/troubleshooting/snapshot/add-repository.asciidoc

```diff
@@ -0,0 +1,11 @@
+[[add-repository]]
+== Mark repository as read-only in other deployments and re-create it in this deployment
```

Comment:
I don't like this title. Any better suggestions?

Reply:
In troubleshooting pages, the title typically denotes the problem that the user is encountering and trying to solve. In which case, maybe something like _Multiple deployments writing to the same repository_ could be a good title.

Reply:
Thanks for the suggestion Adam. I'll make this change.